src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
I know how classes/modules can be nested, I meant that the outer namespace is the Project model class not a different module or class. I though that you meant that Project::UpdatePackagesIfDirty was already inside any other module/class

Ok, anyway. If someone tries to add something wrong to this we should be alert 😄